netsuite partial fetch documentation

tomermevorach opened the pull request.

omrilit commented on packages/netsuite-adapter/specific-cli-options.md:
```diff
@@ -0,0 +1,36 @@
+
+| Name                           | Default when undefined  | Description
+| -------------------------------| ------------------------| -----------
+| netsuite.fetchTarget.filePaths                          | fetch all cabinet files                     | A list of regexes of cabinet file paths. Any file whose path matches any of the regexes will be fetched
```
regexes -> regular expressions?

tomermevorach replied:
fixed

omrilit reviewed (commented):
Is that correct that when you state only `netsuite.fetchTarget.types.[type_name]` you don't **all** fetch the files as well?
It's written in that if you don't provide `netsuite.fetchTarget.filePaths` it fetches all file cabinet content, while in the examples it seems like it doesn't.
I think that some explanation is missing.

alonstern commented:
> Is that correct that when you state only `netsuite.fetchTarget.types.[type_name]` you don't **all** fetch the files as well?
> It's written in that if you don't provide `netsuite.fetchTarget.filePaths` it fetches all file cabinet content, while in the examples it seems like it doesn't.
> I think that some explanation is missing.

That's true. The default of `types` and` file Paths` is to fetch nothing, but the default of the whole `fetchTarget` is to fetch everything. Might be a bit trickier to explain.

tomermevorach commented:
> explanation

tomermevorach commented:
> > Is that correct that when you state only `netsuite.fetchTarget.types.[type_name]` you don't **all** fetch the files as well?
> > It's written in that if you don't provide `netsuite.fetchTarget.filePaths` it fetches all file cabinet content, while in the examples it seems like it doesn't.
> > I think that some explanation is missing.
> 
> That's true. The default of `types` and` file Paths` is to fetch nothing, but the default of the whole `fetchTarget` is to fetch everything. Might be a bit trickier to explain.

Removed the 'default' table column and added an explanation about the default above. Let me know what you think

alonstern commented on packages/netsuite-adapter/specific-cli-options.md:
````diff
@@ -0,0 +1,38 @@
+| Name                           |  Description
+| -------------------------------| ------------------------| -----------
+| netsuite.fetchTarget.filePaths | A list of regular expressions of cabinet file paths. Only files with matching paths will be fetched.
+| netsuite.fetchTarget.types.[type_name]  | A list of script id regular. Only records of type ```type_name``` with matching script ids will be fetched. 
````
```suggestion
| netsuite.fetchTarget.types.[type_name]  | A list of script id regular expressions. Only records of type ```type_name``` with matching script ids will be fetched. 
```

tomermevorach replied:
10x fixed

omrilit reviewed (approved):
I think it's much clearer now!
Maybe you can consider adding one example of using both `filePaths` and `types` options in the same command
